flake.nix

```diff
@@ -23,11 +24,30 @@
 
       isabelle-pkgs = isabelle-nixpkgs.legacyPackages.${system};
 
+        writeShellScriptBinInRepoRoot = name: script: pkgs.writeShellScriptBin name ''
```

Comment:
Can you have these match the indentation? Should be aligned with `isabelle-pkgs`